zerver/lib/user_groups.py

```diff
@@ -78,8 +78,6 @@ def has_user_group_access(
     user_group: NamedUserGroup,
     user_profile: UserProfile,
     *,
-    for_read: bool,
-    as_subgroup: bool,
```

Comment:
`has_user_group_access`  is only used for verifying subgroup access and it only checks deactivated check currently.